samples/bluetooth/central_hids/README.rst

```diff
@@ -207,15 +207,15 @@ Testing with nRF Connect for Desktop
       1. |connect_terminal_specific|
```

Comment:
Shouldn't we change the name of the test section?

```
Testing with nRF Connect for Desktop
```

->

```
Testing with Bluetooth Low Energy app
```

to reflect the change?

That's how I did it in my HIDS mouse sample PR. Should we align the approach?

https://github.com/nrfconnect/sdk-nrf/pull/19039/

Reply:
I see that you also used the new section name in the HIDS keyboard PR

Reply:
This comment is also relevant for the following samples:

- samples/bluetooth/central_and_peripheral_hr/README.rst
- samples/bluetooth/peripheral_power_profiling/README.rst

Reply:
Good catch. Fixed now.

Reply:
thanks, resolving